### PR TITLE
fix event not firing

### PR DESCRIPTION
**Pull request by haileyok**

### CODE REVIEW

File: src/App.native.tsx [hunk 1]
```diff
@@ -168,6 +169,7 @@ function App() {
   const [isReady, setReady] = useState(false)
 
   React.useEffect(() => {
+    setBackgroundColorAsync('#000')
```
Comment (mozzius) [inline src/App.native.tsx:172]: We don't actually need this - it's just that it doesn't get reset to the default black when you reload the dev build. If you reboot the sim it'll be correct

Comment (haileyok) [reply src/App.native.tsx:172]: Ah, so it'll be black in regular builds then? If so then it seems fine to rm this.

Actually, probably preferable. This causes the error screen in the dev client (like for a hard crash) to be black text on black background 🙃

**Review by mozzius — APPROVED** [review]
🔥

**Review by mozzius — CHANGES_REQUESTED** [review]
Actually, found a screen which it doesn't work for

Simulator Screenshot - iPhone 16 - 2024-10-04 at 13 08 01

**Review by mozzius — APPROVED** [review]

**Pull request merged**